cmd/oas-cache/oas-cache.go

```diff
@@ -0,0 +1,73 @@
+// CLI utility that builds cache for OAS file to reduce init time.
```

Comment:
I would like to see more documentation. Maybe a `README.md` with purpose description and usage example?

Reply:
I've added something